intel_extension_for_transformers/llm/runtime/graph/application/common.cpp

```diff
@@ -679,7 +679,7 @@ void quant_print_usage(int argc, char** argv, const quant_params& params) {
   fprintf(stderr, "  --nthread N           number of threads to use (default: 1)\n");
   fprintf(stderr, "  --bits N              number of bits to use for quantization (default: 4)\n");
```

Comment:
py parameter "weight_dtype" and cpp parameter "bits" should be the same

Reply:
Users complained about the parameter difference between Python and cpp.

Reply:
@kevinintel @a32543254

Reply:
how about we change cpp bits to weight_dtype? 
what do you think of it ?